packages/core/src/browser/icons/arrow-down-bright.svg

```diff
@@ -0,0 +1,36 @@
+<!--
+
+    Copyright (c) 2020 Red Hat, Inc.
+    This program and the accompanying materials are made
+    available under the terms of the Eclipse Public License 2.0
+    which is available at https://www.eclipse.org/legal/epl-2.0/
+
+    SPDX-License-Identifier: EPL-2.0
+
+    Contributors:
+      Red Hat, Inc. - initial API and implementation
+
+-->
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
@AndrienkoAleksandr where were these icons taken from?
I don't think we can only use `EPL-2.0`, in general we use:

```
SPDX-License-Identifier: EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0
```

Reply:
I drawn them in the inkscape for this issue, removed not needed tags with some meta information not related to the rendering. If it is not OK, I can try to find similar svg icons in the vscode repo.

Reply:
I think there are errors in the `svg` file, nothing shows up:

![b](https://user-images.githubusercontent.com/40359487/74840654-58e3a300-52f5-11ea-8c50-709da6c6924b.gif)

![error](https://user-images.githubusercontent.com/40359487/74840706-7153bd80-52f5-11ea-84cf-c4084b4b161b.gif)

Reply:
I guess licence header break it.

Reply:
I will take a look.

Reply:
@AndrienkoAleksandr we can use the icons present from the [vscode-icons](https://github.com/microsoft/vscode-icons) repository at the given commit: https://github.com/microsoft/vscode-icons/commit/9c90ce81b1f3c309000b80da0763aa09975a85f0 (the repo is already covered by CQ for this given commit).

There are icons for arrows (the same used by the monaco editor for it's `find`):
 - Ex: [arrow-down](https://github.com/microsoft/vscode-icons/blob/9c90ce81b1f3c309000b80da0763aa09975a85f0/icons/dark/arrow-down.svg)

Reply:
Ok, thanks, I will do it.